RELEASE_NOTES.md

```diff
@@ -0,0 +1,31 @@
+# Release Notes
+## stable-1.15-1.2.0, stable-1.16-1.2.0
+
+* fluent-bit
+  * Disable audit log  collection
+    It's been observed in production clusters that the audit log bloats the number of fields in an index.
+    This causes resource limits to be filled and throttling to occur.
+    We are disabling this collection pending further investigation.
```

Comment:
Just a note, users can still turn this on if they wish to. We are just turning it off by default.